### ISSUE TITLE
Shrink disk on 6 Silver app nodes

### ISSUE THREAD

**Issue by StevenBarre** [BCDevOps/developer-experience#4168]
**Describe the issue**
In an effort to save money, test out using 500G disks on 6 VM workers instead of 800G.

**What is the Value/Impact?**
Reduced costs

**What is the plan? How will this get completed?**
Create CHG and send notifications
Work with VMware team to rebuild the nodes with smaller disks

**Identify any dependencies**
VMware

**Definition of done**
Nodes rebuilt with smaller disks
New ticket created to check performance impacts after the next upgrade

**Comment by StevenBarre** [comment]
CHG0051494 created

**Comment by StevenBarre** [comment]
Postponed due to provincial state of emergency

**Comment by StevenBarre** [comment]
Rescheduled to Oct 4-6th

**Comment by StevenBarre** [comment]
One node (app-63) completed before an incident that impacted the cluster. Will re-schedule and complete later.

**Comment by StevenBarre** [comment]
CHG0054814 scheduled for Jan 22 - Feb 2

**Comment by wmhutchison** [comment]
Mentioned CHG record is approved now. Comms posted in alerts.

**Comment by wmhutchison** [comment]
Unblocked, RFC for this remaining work is now in play. Estimating these specific VMs will be completed by EOD January 23rd 2024.

**Comment by wmhutchison** [comment]
Work completed. Will close off RFC and this ticket tomorrow morning if no issues arise from this work.

**Comment by wmhutchison** [comment]
No issues detected from work, closing off RFC and this ticket, updated comms on alerts channel.
